allennlp_models/modelcards/rc-transformer-qa.json

```diff
@@ -37,8 +38,7 @@
         "model_performance_measures": "F1-score, Span Accuracy, Exact Match. Note that the metrics 
                                        that the model produces are calculated on a per-instance basis only.
                                        Since there could be more than one instance per question, these
-                                       metrics are not the official numbers on the SQuAD task. To get
-                                       official numbers, run the script in scripts/transformer_qa_eval.py.",
```

Comment:
Did the script not survive?

Reply:
It was moved, but it only works for SQuAD 1.1